blockchain/package.json

```diff
@@ -14,7 +14,11 @@
   "license": "ISC",
   "description": "",
   "devDependencies": {
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.11",
     "@nomicfoundation/hardhat-toolbox": "^5.0.0",
     "hardhat": "^2.22.9"
+  },
+  "dependencies": {
+    "ethers": "^6.13.2"
```

Comment:
@Ratawine7 **ethers** is also a peer dependency of **@nomicfoundation/hardhat-toolbox** so no need to install it.